be/src/olap/memtable.h

```diff
@@ -56,7 +61,10 @@ class MemTable {
 
     RowCursorComparator _row_comparator;
     std::unique_ptr<MemTracker> _mem_tracker;
-    std::unique_ptr<MemPool> _mem_pool;
+    // This is a buffer, to hold the memory referenced by the rows that have not
+    // been inserted into the SkipList
+    std::unique_ptr<MemPool> _tmp_mem_pool;
```

Comment:
Would better to explain we add a `_tmp_mem_pool` is in order to make `MemTable::memory_usage` compute accurately.

Reply:
Done.